pkgs/development/python-modules/debugpy/default.nix

```
@@ -1,19 +1,29 @@
-{ stdenv, buildPythonPackage, fetchFromGitHub
-, substituteAll, gdb
-, colorama, django, flask, gevent, psutil, pytest
-, pytest-timeout, pytest_xdist, requests
+{ stdenv
```

Same recommendation as in #99216, loose the stdenv.lib indirection.

```suggestion
{ lib
```